graphqlapi/pom.xml

```diff
@@ -104,10 +104,6 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-servlets</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-server</artifactId>
-    </dependency>
```

Comment:
so this is not needed or what?

Reply:
It's a duplicate of a dependency already listed above. The dependency checker chokes on it.